frontends/mit-open/src/pages/AboutPage/AboutPage.tsx

```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
               ancestors={[{ href: urls.HOME, label: "Home" }]}
               current="About Us"
             />
-            <Header variant="h3">About Us</Header>
+            <Typography variant="h3" component="h1">
+              About Us
+            </Typography>
           </BannerContainerInner>
         </BannerContainer>
         <BodyContainer>
           <MissionStatementContainer>
-            <MissionStatementHeader>Mission</MissionStatementHeader>
-            <BodyText variant="body1">
-              MIT's mission to share learning with the world has been central to
-              the Institute since its founding in 1861. At the start of the 21st
-              century, MIT took that mission online, bringing learning resources
-              to a broader audience through digital platforms. Today, MIT is
-              advancing this mission further by providing non-degree learning
-              resources &ndash; online, in-person, and blended courses and
-              programs that offer the opportunity to learn from MIT faculty,
-              industry experts, and a global community of learners without
-              enrolling in a degree-seeking program. Through these resources,
-              millions of learners have gained the knowledge and tools needed to
-              advance their academic and professional goals.
-            </BodyText>
+            <Typography variant="body1">
+              Since its founding in 1861, MIT has been committed to sharing
+              knowledge with the world. At the beginning of the 21st century,
+              MIT expanded this mission by bringing learning resources online,
+              reaching a broader audience through digital platforms. Today, MIT
+              continues to advance this mission by offering non-degree learning
+              resources, including online, in-person, and blended courses and
+              programs. These opportunities allow learners to study with MIT
+              faculty, industry experts, and a global community without
+              enrolling in a degree program. Through these resources, millions
+              of learners have acquired the knowledge and skills needed to
+              further their academic and professional goals.
+            </Typography>
           </MissionStatementContainer>
           <SubHeaderContainer>
             <SubHeaderTextContainer>
               <Typography variant="subtitle1">
                 Our non-degree learning resources empower you to:
               </Typography>
-              <ul>
+              <List>
```

Comment:
Can we put the mission statement here, above the list and alongside the image, but without the `MissionStatementContainer`?